diff --git a/src/pages/_app.tsx b/src/pages/_app.tsx
index a0cc1e6..cdd5387 100644
--- a/src/pages/_app.tsx
+++ b/src/pages/_app.tsx
@@ -1,6 +1,6 @@
 import { clsx } from 'clsx';
 import { type AppType } from 'next/app';
-import { Poppins } from 'next/font/google';
+import localFont from 'next/font/local';
 import Head from 'next/head';
 import { useRouter } from 'next/router';
 import { type Session } from 'next-auth';
@@ -20,7 +20,40 @@ import { useAppStore } from '~/store/appStore';
 import { type NextPageWithUser } from '~/types';
 import { api } from '~/utils/api';
 
-const poppins = Poppins({ weight: ['200', '300', '400', '500', '600', '700'], subsets: ['latin'] });
+const poppins = localFont({
+  src: [
+    {
+      path: '../../public/fonts/Poppins-ExtraLight.ttf',
+      weight: '200',
+      style: 'normal',
+    },
+    {
+      path: '../../public/fonts/Poppins-Light.ttf',
+      weight: '300',
+      style: 'normal',
+    },
+    {
+      path: '../../public/fonts/Poppins-Regular.ttf',
+      weight: '400',
+      style: 'normal',
+    },
+    {
+      path: '../../public/fonts/Poppins-Medium.ttf',
+      weight: '500',
+      style: 'normal',
+    },
+    {
+      path: '../../public/fonts/Poppins-SemiBold.ttf',
+      weight: '600',
+      style: 'normal',
+    },
+    {
+      path: '../../public/fonts/Poppins-Bold.ttf',
+      weight: '700',
+      style: 'normal',
+    },
+  ],
+});
 const toastOptions = { duration: 1500 };
 
 const MyApp: AppType<{ session: Session | null; baseUrl: string }> = ({
